File: IsometricShooter/Scripts/CS/Class/UnitSpawnClass.cs
using UnityEngine;
using System;
using System.Collections;

[Serializable]
public class UnitSpawn {

	public GameObject unit; //any GameObject
	public Transform spawnPoint; //position of spawn point
	public float spawnRate = 2.0f; //spawn rate in seconds
	public int amount = 1; //amount of units to be spawned, 0 = no unit

}

File: IsometricShooter/Scripts/JS/Class/UnitSpawnClass.js
#pragma strict

class UnitSpawn {

	var unit : GameObject; //any GameObject
	var spawnPoint : Transform; //position of spawn point
	var spawnRate : float = 2.0; //spawn rate in seconds
	var amount : int = 1; //amount of units to be spawned, 0 = no unit

}